app/gateway/changelog.md

```diff
@@ -308,6 +308,19 @@ Kong Gateway version.
 * The function `kong.db.*.cache_key()` now falls back to `.id` if nothing from `cache_key` is found.
   [#8553](https://github.com/Kong/kong/pull/8553)
 
+### Known limitations
+
+* Kong Manager does not currently support the following features:
+  * Secrets management
+  * Plugin ordering
+  * Expression-based routing
+
+* Blue-green deployments:
+  * **Kong Gateway (OSS)**: Upgrades with blue-green deloyments are not supported for major versions, therefore they are not supported with upgrades from 2.x.x to 3.0.x.
+  * **Kong Enterprise**: Blue-green upgrades from versions 2.1.x.x-2.7.x.x to 3.0.0.0 are not currently supported.
```

Comment:
Well I clearly misunderstood the situation.

Reply:
Now I'm asking to ensure I have the correct understanding too 😓

Reply:
There are 2 kinds of migration that has no downtime.
1. For a hybrid mode deployment with no version compatibility between versions, we could first upgrade to a new CP, start new DPs, and gracefully stop old DPs. During the process old DPs will not be able to accept new config;
2. For both modes (hybrid and traditional), next 2.8.x.x of EE supports blue/green migration so the old instances can run on the migrated database. But it may cause problems when you change the config with an old instance via admin API for traditional mode.

However, I don't know whether the first kind of migration should be called blue/green.

Reply:
Confirmed that this is the blue/green migration. Going to submit the modification.